brainglobe_atlasapi/atlas_generation/atlas_scripts/praire_vole.py

```diff
@@ -0,0 +1,256 @@
+__version__ = "0"
+
+import os
+import random
+import shutil
+import time
+import zipfile
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pooch
+import tifffile
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir):
+    ATLAS_NAME = "..."
+    SPECIES = "..."
+    ATLAS_LINK = "https://elifesciences.org/articles/87029#data"
+    CITATION = "..."
+    ATLAS_BASE_URL = "https://ndownloader.figshare.com/files/"
+    ORIENTATION = "..."
+    ROOT_ID = 997
+    ATLAS_PACKAGER = "..."
+    ADDITIONAL_METADATA = {}
+
+    # setup folder for downloading
+    download_dir_path = working_dir / "downloads"
+    download_dir_path.mkdir(exist_ok=True)
+    atlas_path = download_dir_path / "atlas_files"
+
+    files = {
+        "PrV_ReferenceBrain_bilateral.tif": "42917449",
+        "PrV_Annotation_bilateral.tif": "42917452",
+        "PrV_Atlas_Ontology.csv": "42917443",
+        "PrV_results_datasets.zip": "42501327",
+    }
+
+    utils.check_internet_connection()
+    pooch.create(
+        path=atlas_path,
+        base_url=ATLAS_BASE_URL,
+        registry={},
+    )
+
+    for filename, file_id in files.items():
+        url = f"{ATLAS_BASE_URL}{file_id}"
+        pooch.retrieve(
+            url=url,
+            fname=filename,
+            path=atlas_path,
+            known_hash=None,
+        )
+
+    with zipfile.ZipFile(
+        r"C:\Users\sacha\Downloads\atlas shit\PrV_results_datasets.zip", "r"
```

Comment:
This should be a relative path, defined by Pathlib so it will work on any system, e.g. `atlas_path / "PrV_results_datasets.zip"`